### ISSUE TITLE
Unclear message when there is no tag present for the package's version

### ISSUE THREAD

**Issue by deshipu** [softwarefactory-project/rdopkg#106]
When the version in the spec file is set to a value that doesn't have a corresponding tag in the repositories, rdopkg will display this message:
```
$ rdopkg patch
## get_package_env
RHOS package detected.
## ensure_patches_branch
## get_patches_branch
git branch -f rhos-11.0-patches rhos/rhos-11.0-patches
## check_new_patches
command failed: git log --format=%h%x1f%s%x1f%b%x1e 11.0.1..rhos/rhos-11.0-patches
stderr:
fatal: ambiguous argument '11.0.1..rhos/rhos-11.0-patches': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

No new patches detected in rhos/rhos-11.0-patches.

Do you want to continue anyway? [Ny]
```

It would be nicer to display a more specific error message in that case, such as:
```
Package is at Version: 1.2.3 but 1.2.3 tag isn't present.
```

**Comment by yac** [comment]
Review underway, there are some unwanted sideeffects... https://review.rdoproject.org/r/#/c/5921/

**Comment by ssbarnea** [comment]
I don't think that prompting for input on non-tty would ever be attempted. That's clearly a serious bug that affects CI usage.

**Comment by yac** [comment]
This was addressed by 74ee70e1 , nice error message is presented now.
